pkgs/applications/emulators/ruffle/default.nix

```diff
@@ -9,27 +9,39 @@
 , wayland
 , xorg
 , vulkan-loader
+, jre_minimal
+, cairo
+, gtk3
+, wrapGAppsHook
+, gsettings-desktop-schemas
+, glib
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "ruffle";
-  version = "nightly-2022-02-02";
+  version = "nightly-2022-09-26";
 
   src = fetchFromGitHub {
     owner = "ruffle-rs";
     repo = pname;
     rev = version;
-    sha256 = "sha256-AV3zGfWacYdkyxHED1nGwTqRHhXpybaCVnudmHqWvqw=";
+    sha256 = "o0geKXODFRPKN4JgW+Sg16uPhBS5rrlMCmFSc9AcNPQ=";
```

Comment:
```suggestion
    hash = "sha256-o0geKXODFRPKN4JgW+Sg16uPhBS5rrlMCmFSc9AcNPQ=";
```

Reply:
Key doesn't really *need* to be changed, though using a bare base64 hash is definitely uncommon.

Maybe we should just merge this until the whole hash format/key issue is sorted out? Or at least change to SRI but keep the key? /shrug

Reply:
the sha256- prefix should be mainly kept because otherwise the hash is broken when sha256 is changed to hash.

Reply:
I went half-way and re-added the prefix. I did not make any intentional changes, I just ran `nix-prefetch-github` with the `--nix` option, and that's what it output. If it matters significantly, it might be a good idea to have this changed in that tool, since I am sure I am not the only person using it to contribute to nixpkgs.